pkgs/top-level/all-packages.nix

```diff
@@ -11763,7 +11763,7 @@ with pkgs;
 
   darwin = let
     apple-source-releases = callPackage ../os-specific/darwin/apple-source-releases { };
-  in apple-source-releases // rec {
+  in recurseIntoAttrs (apple-source-releases // rec {
```

Comment:
That doesn't prevent those packages from getting built & redistributed via indirect dependencies. E.g. `vim_configurable` depends on `cf-private`, so that just looks like a no-op here.